command/server/config_test_helpers.go

```diff
@@ -503,7 +503,7 @@ func testUnknownFieldValidation(t *testing.T) {
 			Problem: "unknown or unsupported field bad_value found in configuration",
 			Position: token.Pos{
 				Filename: "./test-fixtures/config.hcl",
-				Offset:   651,
+				Offset:   652,
```

Comment:
Same as in the other review, is this change intentional?

Reply:
Yup it was intentional, the test code is leveraging the file position markers within command/server/test-fixtures/config.hcl so it needed to be updated. The same fix was done on 1.15/main